init_setup.sh

```diff
@@ -16,19 +16,25 @@ dbname="FileFighterDB"
 networkname="FileFighterNetwork"
 
 # latest stable versions.
-frontendVersion="latest" #"$(getTagsByName filefighter/frontend v | tail -1)"
-restVersion="latest" #$(getTagsByName filefighter/rest v | tail -1)"
+frontendVersion="latest"
+restVersion="latest"
 
 # Startup Message.
+
+echo "  _____   _   _          _____   _           _       _                 "
+echo " |  ___| (_) | |   ___  |  ___| (_)   __ _  | |__   | |_    ___   _ __ "
+echo " | |_    | | | |  / _ \ | |_    | |  / _\` | | '_ \  | __|  / _ \ | '__|"
+echo " |  _|   | | | | |  __/ |  _|   | | | (_| | | | | | | |_  |  __/ | |   "
+echo " |_|     |_| |_|  \___| |_|     |_|  \__, | |_| |_|  \__|  \___| |_|   "
+echo "                                     |___/                             "
+echo "                   Version 1.2 Last updated at 29.10.20                "
```

Comment:
version could a variable maybe? So you don't have to change it in every file?

Reply:
so could date, yes.